service/SpaceCenter/src/Services/Orbit.cs

```diff
@@ -362,5 +362,138 @@ public double OrbitalSpeedAt (double time)
         {
             return InternalOrbit.getOrbitalSpeedAt (time);
         }
+
+        /// <summary>
+        /// The orbital radius at the given time, in meters.
+        /// </summary>
+        /// <param name="time">Time from now, in seconds.</param>
+        [KRPCMethod]
+        public double RadiusAt(double time)
+        {
+            return InternalOrbit.getRelativePositionAtUT(time).magnitude;
+        }
+
+        /// <summary>
+        /// The orbital radius at the given time, in meters.
+        /// </summary>
+        /// <param name="time">Time from now, in seconds.</param>
+        /// <param name="rf">Reference Frame.</param>
+        [KRPCMethod]
+        public Tuple3 PositionAt(double time, ReferenceFrame rf)
+        {
+            return rf.PositionFromWorldSpace(InternalOrbit.getPositionAtUT(time)).ToTuple();
+
+        }
+
+
+        /// <summary>
+        /// Estimates time of closest approach in the next orbit.
+        /// </summary>
+        /// <param name="target">Other vessel.</param>
+        [KRPCMethod]
+        public double ClosestApproach(Vessel target)
+        {           
+            Orbit b = target.Orbit;
```

Comment:
Rename this variable to something like "targetOrbit" so that it's self documenting.
(Also you can drop the type and just put "var targetOrbit = target.Orbit;" and similar in various places in the rest of the code.)